ibis/expr/api.py

```diff
@@ -1344,6 +1345,11 @@ def clip(arg, lower=None, upper=None):
     return op.to_expr()
 
 
+def random():
```

Comment:
let's create a ibis.expr.random; the goal is to produce a random scalar yes? reference the documentation https://docs.python.org/3/library/random.html, which i assume is the goal here.